internal/accounts/create.go

```diff
@@ -256,21 +247,27 @@ func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow
 	if err != nil {
 		return nil, err
 	}
-	log.Info(fmt.Sprintf("\n %s Successfully created account on %s", output.OkEmoji(), networkName))
-	output.NextStepPrompt()
+
+	log.Info(fmt.Sprintf(
+		"%s New account created with address: %s, and name: %s",
```

Comment:
```suggestion
		"%s New account created with address 0x%s and name %s",
```

No need for the extra punctuation IMO.